insights/parsers/smartctl.py

```diff
@@ -202,3 +211,51 @@ def parse_attributes(line):
 
         # Delete temporary full line storage
         del self.full_line
+
+
+@parser(Specs.smartctl_l_scterc)
+class SMARTctlSCTERC(CommandParser, dict):
+    """
+    Parser for output of ``smartctl -l scterc`` for each drive in system.
+
+    This stores the SCT ERC (Smart Command Transfer Error Recovery Control) information
+    from the output of `smartctl -l scterc` in the following properties:
+    following properties:
+
+    * ``device`` - the name of the device after /dev/ - e.g. sda
+
+    Sample output::
+
+        smartctl 7.1 2020-04-05 r5049 [x86_64-linux-4.18.0-240.el8.x86_64] (local build)
+        Copyright (C) 2002-19, Bruce Allen, Christian Franke, www.smartmontools.org
+        SCT Error Recovery Control set to:
+         Read: 200 (20.0 seconds)
+         Write: 200 (20.0 seconds)
+
+    Examples:
+        >>> scterc.device
+        '/dev/sda'
+        >>> scterc['Read']
+        20.0
+        >>> scterc['Write']
+        20.0
```

Comment:
@shlao since you only have the two keys "Read" and "Write", why not just make those 2 attributes of the class instead of using a `dict` class here?  Are you anticipating additional keys in the future?